### PR TITLE
869993: committed new content for the volume 1 release.

### PR DESCRIPTION
**Pull request by indumathi-murali**

### CODE REVIEW

**Comment by indumathi-murali** [comment]
Tested sample link - 
https://www.syncfusion.com/downloads/support/directtrac/general/ze/MapsCore738756158
https://www.syncfusion.com/downloads/support/directtrac/general/ze/MVCMAps-1982701592

File: ej2-asp-core-mvc/code-snippet/maps/polygon/polygon-shape/tagHelper [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+        new { longitude = 36.100009274845206, latitude = 29.15308642012721 },
+        new { longitude = 34.85774476486728, latitude = 29.3103032832622 },
+        new { longitude = 34.64498583263142, latitude = 28.135787235699823 },
+        new { longitude = 34.88539587371454, latitude = 28.181421087099537 }
+    };
+
+
+    var polygons = new List<Syncfusion.EJ2.Maps.MapsPolygon>
+    {
+        new MapsPolygon{ Points=data, Fill="blue", Opacity=0.7, BorderColor="green", BorderOpacity=0.7, BorderWidth=2 }
```
Comment (sabari-senthamaraikannan) [inline ej2-asp-core-mvc/code-snippet/maps/polygon/polygon-shape/tagHelper:56]: tooltipTemplate is not available in this code.

File: ej2-asp-core-mvc/code-snippet/maps/polygon/polygon-shape/razor [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+        new { longitude = 36.100009274845206, latitude = 29.15308642012721 },
+        new { longitude = 34.85774476486728, latitude = 29.3103032832622 },
+        new { longitude = 34.64498583263142, latitude = 28.135787235699823 },
+        new { longitude = 34.88539587371454, latitude = 28.181421087099537 }
+    };
+
+
+    var polygons = new List<Syncfusion.EJ2.Maps.MapsPolygon>
+{
+        new Syncfusion.EJ2.Maps.MapsPolygon{ Points=data, Fill="blue", Opacity=0.7, BorderColor="green", BorderOpacity=0.7, BorderWidth=2 }
```
[the remark this answers is not in the document]
Comment (indumathi-murali) [reply ej2-asp-core-mvc/code-snippet/maps/polygon/polygon-shape/razor:56]: This is not the code related to tooltipTemplate. Previous missed code snippets are moved here.

File: ej2-asp-core-mvc/code-snippet/maps/polygon/polygon-shape/tagHelper (hunk 1 above)
Comment (indumathi-murali) [reply ej2-asp-core-mvc/code-snippet/maps/polygon/polygon-shape/tagHelper:56]: This is not the code related to tooltipTemplate. Previous missed code snippets are moved here.

**Review by sabari-senthamaraikannan — APPROVED** [review]

**Pull request merged**